cumulus/bridges/rustfmt.toml

```diff
@@ -1,24 +0,0 @@
-# Basic
```

Comment:
@serban300 
could you, please, exclude this file from `verify-pallets-build.sh` in bridges repo?
Just to avoid bringing it back on the next subtree update.

Anyway, I will file a issue soon for moving bridges to the monorepo and move subtree from cumulus subdirectory out.

Reply:
Good point. Will do.